packages/headless/src/api/search/search/product.ts

```diff
@@ -1,15 +1,82 @@
 export type Product = {
+  /**
+   * The SKU of the product.
+   */
   sku: string;
+  /**
+   * Name of the product.
+   *
+   * From the `ec_name` field.
+   */
   name: string;
-  thumbnailUrl: string;
+  /**
+   * Description of the product.
+   *
+   * From the `ec_description` field.
+   */
+  description: string;
+  /**
+   * A direct link to the product in URL format.
+   */
   link: string;
-  price: number;
-  promoPrice: number;
-  rating: number;
-  tags: string[];
-  brand: string;
-  categories: string[];
-  inStock: boolean;
+  /**
+   * Product brand.
+   *
+   * From the `ec_brand` field.
+   */
+  brand?: string;
+  /**
+   * Category of the product (e.g., `["Electronics", "Electronics|Televisions", "Electronics|Televisions|4K Televisions"]`)
+   *
+   * From the `ec_category` field.
+   */
+  categories?: string[];
+  /**
+   * Base price of the product or variant.
+   *
+   * From the `ec_price` field.
+   */
+  price?: number;
+  /**
+   * Short description of the product.
+   *
+   * From the `ec_shortdesc` field.
+   */
+  shortDescription?: string;
+  /**
+   * Product image in URL format.
+   *
+   * From the `ec_image` field.
+   */
+  thumbnailUrl?: string;
+  /**
+   * Product images in URL format.
+   *
+   * From the `ec_images` field.
+   */
+  imageUrls?: string[];
```

Comment:
The descriptions here are almost identical, do you think we should be more explicit with the descriptions?

Reply:
Good point, what about `Additional product images in URL format.`?

Reply:
Definitely an improvement 👍🏼